docs/lib/render-code.js

```diff
@@ -0,0 +1,13 @@
+module.exports = renderCode;
+
+const prism = require('prismjs');
+prism.languages.html = require('./prism-twig'); // use twig (same syntax as nunjucks) as default formatter for html code
+
+function renderCode(code, language) {
+	if (prism.languages.hasOwnProperty(language)) {
```

Comment:
if language is `jinja`, `nunjucks` or `njk` set it to `twig` so it's properly highlighted.